### ISSUE TITLE
backspace not working in terminal

### ISSUE THREAD

**Issue by imkzh** [flozz/nautilus-terminal#19]
backspace is used as go back in nautilus, and each time i press backspace in terminal, the folder changes and command was submitted followed by a ` cd some_dir`.
(same as #8) 
```
$ uname -a
Linux MyUbuntu 4.4.0-134-generic #160-Ubuntu SMP Wed Aug 15 14:58:00 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

$ python --version
Python 2.7.12

$ python3 --version
Python 3.5.2
```

**Comment by flozz** [comment]
Hello,

* How did you install Nautilus Temrinal? (from pypi? from git?)
* What is you Ubuntu version (`cat /etc/issue`)

**Comment by imkzh** [comment]
i install Nautilus Temrinal from git (by `pip install .`)
and my ubuntu is `Ubuntu 16.04.5 LTS \n \l` reported by `cat /etc/issue`

**Comment by flozz** [comment]
Ok, I will try to reproduce the issue in a VM this weekend, thanks for the quick answer :)

**Comment by flozz** [comment]
Hello,

I was able to reproduce your issue on a Ubuntu 16.04.5 VM, you cannot use any Nautilus accel in the terminal (backspace, /, ~, ctrl+d,...)... And when I look at the console when starting the extension in debug mod, it will not be the only problem.

Nautilus Terminal is a big hack on the top of the Nautilus Extension system. It is very complicated to just make it work for recent Nautilus version and to maintain it working on newer version.

To support Nautilus 3.14 (the version of Nautilus available on Ubuntu 16.04, that was already an old Nautilus version when Ubuntu 16.04 was released), an almost complete rewrote of the extension is necessary. I will not have time to do this, sorry :(

Your only choices are:

* updating to Ubuntu 18.04 LTS, 
* using Dolphin (the KDE file browser, that is equivalent to Nautilus and that have natively an integrated terminal)

Sorry to not being able to help more with this :(

**Comment by imkzh** [comment]
My research is heavily relied on Ubuntu and many thrird party packages were built just for 16.04, so it is not easy for me to migrate to a newer version or some new system.

Anyway, i will miss this great project, and thank you for your reply :)
